src/contractTest/resources/application.properties

```diff
@@ -1,3 +1,4 @@
 idam.url=http://localhost:8892
 document_management.url=${DM_STORE_URL:http://dm-store}
-core_case_data.api.url=http://localhost:8891
\ No newline at end of file
+core_case_data.api.url=http://localhost:8891
+core_case_data.api.search.url=http://localhost:8891
```

Comment:
Why do you add `core_case_data.api.search.url` this has same value with `core_case_data.api.url`

Reply:
In this test, CcdClient is used in test. Without this parameter, spring cannot create a feign client bean for CcdClient
`
@FeignClient(
    name = "ccd-client",
    url = "${core_case_data.api.search.url}"
)
@SuppressWarnings("PMD.UseObjectForClearerAPI")
public interface CcdClient {`